pyproject.toml

```diff
@@ -35,7 +35,8 @@ classifiers = [
 
 [tool.poetry.dependencies]
 python = "^3.6.0"
```

Comment:
I suggest to keep this list sorted for consistency

Reply:
Done